ramls/loan.json

```diff
@@ -103,6 +103,11 @@
       "type": "string",
       "format": "date-time"
     },
+    "agedToLostDate": {
+      "description": "Date and time the item was aged to lost for this loan",
+      "type": "string",
+      "format": "date-time"
+    },
```

Comment:
Is this information needed for processes other than issuing fees / fines and refunds as part of ageing an item to lost?

Reply:
No, currently no other cases.

Reply:
Then could it be part of the temporary `agedToLostDelayedBilling` object instead?

Reply:
Yes, it can be, let me move it to that object.

Reply:
Done